wandb/sdk/internal/internal_api.py

```diff
@@ -3569,7 +3569,7 @@ def set_sweep_state(
             raise Exception("Cannot resume %s sweep." % curr_state.lower())
         elif state == "PAUSED" and curr_state not in ("PAUSED", "RUNNING"):
             raise Exception("Cannot pause %s sweep." % curr_state.lower())
-        elif curr_state not in ("RUNNING", "PAUSED"):
+        elif curr_state not in ("RUNNING", "PAUSED", "PENDING"):
```

Comment:
you should be able to set state to "CANCELLED" when the sweep state is "PENDING" right?

Reply:
This is true in the frontend, but does **not** work from the CLI.... This change corrects that discrepancy.

Reply:
Nice, what about vanilla sweeps do they allow it?

Reply:
No, this API path is for both vanilla and launch, so we would be adding the ability to set state for both launch and vanilla sweeps from "pending" to "cancelled". IN the UI we can do this for both vanilla and launch currently. @KyleGoyette